targets/TARGET_ARM_SSG/TARGET_CM3DS_MPS2/device/TOOLCHAIN_IAR/MPS2.icf

```diff
@@ -0,0 +1,56 @@
+/*
+ * MPS2 CMSIS Library
+ */
+/*
+ * Copyright (c) 2009-2017 ARM Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License) you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+/* The RAM region doesn't start at the beginning of the RAM address 
+ * space to create space for the vector table copied over to the RAM by mbed.
+ * The space left is a bit bigger than is necessary based on the number of 
+ * interrupt handlers. 
+ */
+/*###ICF### Section handled by ICF editor, don't touch! ****/
+/*-Editor annotation file-*/
+/* IcfEditorFile="$TOOLKIT_DIR$\config\ide\IcfEditor\cortex_v1_0.xml" */
+/*-Specials-*/
+define symbol __ICFEDIT_intvec_start__ = 0x00000000;
+/*-Memory Regions-*/
+define symbol __ICFEDIT_region_ROM_start__    = 0x00000000;
+define symbol __ICFEDIT_region_ROM_end__      = 0x0003FFFF;
+define symbol __ICFEDIT_region_RAM_start__    = 0x20000140;
+define symbol __ICFEDIT_region_RAM_end__      = 0x2001FFFF;
+/*-Sizes-*/
+/* Heap and Stack size */
+define symbol __ICFEDIT_size_cstack__ = 0x4000;
```

Comment:
why cstack is much more bigger than heap? For mbed OS 5 this is value used for isr stack, this value is way too big.  and heap too small then